database/snowflake/README.md

```diff
@@ -0,0 +1,4 @@
+# Integration with Snowflake using User-Defined Functions
+CADP-SNOW-AWS-Functionsis for AWS Lambda 
```

Comment:
we would need to expand on the README for the projects -

- like provide usage example
- compilation and installation instructions
- how to setup in a snowflake environment